### DISCUSSION TITLE
TMS coil location 4mm above the scalp

### DISCUSSION THREAD

**Discussion by eburkevt** [simnibs/simnibs#419] [2. General]
I found in an old discussion [https://github.com/simnibs/simnibs/discussions/199](url) that  ... 

    "if matsimnibs is provided then pos.distance is not used because the [TMS] coil position is fully defined by matsiminibs."

So if we specify poslist{1,1}.pos.matsimnibs [4x4 matrix, matlab], then poslist.pos.distance (usually =4 mm?) is not applied (i.e. is ignored)? 

I assume that means we must ensure that the 4th column, rows 1-3, of poslist.pos.matsimnibs matrix represents the point 4 mm  above, and normal to, the scalp surface, which is where the TMS coil location is simulated?  We can do this calculation in matlab, but just wanted to ensure that this is the correct coil position.

Thanks for your help!

**Comment by TorgeW** [comment]
Yes if matsimnibs is set, distance is ignored. 
This is because the affine transformation gives you full control of the location and direction so you have to ensure that the location is wherever you want it to be.

**Reply by eburkevt** [reply]
Great, thank you!
